portfolio/src/main/java/com/google/sps/data/LogObject.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A comment message. */
+public final class LogObject {
+
+  private boolean loginStatus = false;
+  private String loginUrl = null;
+  private String logoutUrl = null;
+  private String userEmail = null;
+  private String username = null;
+
+  public LogObject() {
```

Comment:
Understand this is a data type, and it's only used to convoy data between server and browser. I think it is OK to use public accessibility for all its members, that way, you don't have to create setter/getter functions. It's common for pure data classes to always use public members.